google-cloud-spanner-hibernate-samples/basic-spanner-features-sample/pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <artifactId>google-cloud-spanner-hibernate-samples</artifactId>
     <groupId>com.google.cloud</groupId>
-    <version>1.5.4</version>
+    <version>0.0.1-SNAPSHOT</version><!-- This artifact should not be released  -->
```

Comment:
Let's keep 1.5.4 with the instruction to release infrastructure to keep it up to date (`<!-- {x-version-update:google-cloud-spanner-hibernate:current} -->` for now. We'll switch to 0.0.1 at some later point.